README_JP.md

```diff
@@ -0,0 +1,290 @@
+    ただし、このメソッドの実行前に BeginGDI() メソッド、実行後に EndGDI() メソッドを実行する必要があることに注意してください。
+  
+  - GDIHelper.FillRect12(*color*, *x1*, *y1*, *x2*, *y2*)
+    GDIHelper.FillRectWH(*color*, *x1*, *y1*, *width*, *height*)
+
+    単一色で塗り潰された長方形を描画します。
+    このメソッドの実行前に BeginGDI() メソッド、実行後に EndGDI() メソッドを実行する必要があることに注意してください。
+  
+  BeginGDI() メソッドを実行してから EndGDI メソッドを実行するまでの間に GDIHelper.Graphics を使用しないでください。
+  また、BeginGDI() メソッドを実行してから EndGDI メソッドを実行するまでの間以外で GDIHelper.DrawImage メソッド、GDIHelper.FillRect から始まる名前のメソッドを使用しないでください。
+  
+- TextureHandle.ShouldUpdate(*time*, *fps*)
+
+  `time` で指定した現在時刻を基に、`fps` で指定したフレームレート (テクスチャの更新頻度) の下限を下回っているかを取得します。
+  テクスチャの更新はコストが大きいため、できるだけ頻度を下げることが推奨されます。
+
+  このメソッドは単に「最後の更新からどのくらいの時間が経過したか」を確認しているのみなので、このメソッドを使わず、強制的に更新することもできます。
+
+- TextureHandle.Update(*gdiHelper or bitmap*)
+
+  ビデオカードに GDIHelper や Bitmap をプッシュし、TextureHandle に紐づけられたゲーム内のテクスチャを更新します。
+  テクスチャの更新はコストが大きいため、更新頻度の最適化のために TextureHandle.ShouldUpdate メソッドを使用することを推奨します。
+
+- TextureManager.Dispose()
+
+  プラグインのアンロードの際は、全リソースの解放を忘れないようにしてください。
+
+## タッチ
+
+コード例 (重要な部分のみであり、完全なコードではありません) :
+
+```csharp
+// using System;
+// using System.Windows.Forms;
+// using Zbx1425.DXDynamicTexture;
+
+private static TouchTextureHandle hTIMSTex;
+
+[DllExport(CallingConvention.StdCall)]
+public static void Load(){
+    TextureManager.Initialize();
+    hTIMSTex = TouchManager.Register("foo/bar/tims_placeholder.png", 512, 512);
+    hTIMSTex.SetClickableArea(0, 0, 400, 300);
+    TouchManager.EnableEvent(MouseButtons.Left, TouchManager.EventType.Down);
+    hTIMSTex.MouseDown += HTIMSTex_MouseDown;
+}
+
+private static void HTIMSTex_MouseDown(object sender, TouchEventArgs e) {
+    MessageBox.Show(String.Format("X: {0}, Y: {1}", e.X, e.Y));
+}
+```
+
+動的テクスチャのコードとほぼ同じ処理になります。相違点としては TouchManager.Register メソッドを使用することと、同メソッドの戻り値が TouchTextureHandle 型であることくらいです。
+TouchTextureHandle は TextureHandle を継承しているため、動的テクスチャのための機能は TouchTextureHandle にも含まれます。
+
+また、同一のファイルを TouchManager と TextureManager の両方に登録しないでください。TouchManager.Register メソッドには TextureManager.Register メソッドと同一の機能が含まれます。
+
+また、タッチされた位置を色で検出するため、CabIlluminance による運転台の明るさの設定には対応しないことをご了承ください。DaytimeImage と NighttimeImage で同一のファイルを指定してください。
+
+- TouchTextureHandle.SetClickableArea(*x0*, *y0*, *width*, *height*)
+
+  テクスチャ内の一部の領域のみをタッチ可能に設定します。このメソッドが呼び出されなかった場合はテクスチャ内の全領域がタッチ可能に設定されます。
+
+- TouchManager.EnableEvent(*button*, *type*)
+
+  イベントを発生させる条件を指定します。
+  例えば、`TouchManager.EnableEvent(MouseButtons.Left, TouchManager.EventType.Down);` は左マウス ボタンが押されたときにのみイベントを発生させるように設定します。
+
+  タッチされた位置を一瞬グラデーションを表示して検出するという本ライブラリの仕様上、全てのマウス動作に対してイベントを発生させると高頻度な点滅が発生することとなり、見栄えが悪くなってしまいます。
+  これを防ぐために、このメソッドでイベントを発生させる条件を指定することができるようにしています。
+
+  MouseButtons を使用するには、参照に `System.Windows.Forms` を追加してください。
+
+- TouchTextureHandle.MouseDown += *handler*, TouchTextureHandle.MouseUp += *handler*
+
+  画面にタッチされたときに発生するイベントを購読します。
+
+  `e.X`、`e.Y` は、タッチ可能な領域の左上端を原点としたマウスの位置をピクセル単位で表します。
```

Comment:
I edited the English document to mention that `e.X` and `e.Y` are in the pixel unit related to the size when registering `TouchManager`.

Reply:
Added the same content as README.md in [aa9e8b8](https://github.com/zbx1425/DXDynamicTexture/pull/1/commits/aa9e8b8f5bd909209b819af70bf0b5aa5f38a44f).